K2Bridge/Visitors/MetricAggregationVisitor.cs

```diff
@@ -39,5 +39,14 @@ public void Visit(MaxAggregation maxAggregation)
 
             maxAggregation.KustoQL = $"{maxAggregation.FieldAlias}={KustoQLOperators.Max}({maxAggregation.FieldName})";
         }
+
+        /// <inheritdoc/>
+        public void Visit(SumAggregation sumAggregation)
+        {
+            Ensure.IsNotNull(sumAggregation, nameof(sumAggregation));
+            EnsureClause.StringIsNotNullOrEmpty(sumAggregation.FieldName, sumAggregation.FieldName, ExceptionMessage);
+
+            sumAggregation.KustoQL = $"{sumAggregation.FieldAlias}={KustoQLOperators.Sum}({sumAggregation.FieldName})";
```

Comment:
I would recommend quoting all field aliases to avoid problems with characters that are illegal in Kusto (e.g. the dash "-").

```suggestion
            sumAggregation.KustoQL = $"['{sumAggregation.FieldAlias}']={KustoQLOperators.Sum}({sumAggregation.FieldName})";
```

Reply:
Fixed.